src/routes/tools/streamflow/_helpers.js

```diff
@@ -0,0 +1,16 @@
+import getCenter from "@turf/center";
+import { MONTHS_LIST } from "~/routes/tools/_common/constants";
+
+export const getSelectedMonthStrings = (monthIdsArr) => {
+  const idSet = new Set(monthIdsArr);
+  if (idSet.size) {
+    return MONTHS_LIST.filter(({ id }) => idSet.has(id)).map(
+      ({ text }) => text
+    );
+  }
+};
+
+export function getBasinCenter(feature) {
+  const { geometry } = getCenter(feature.geometry);
```

Comment:
it could be good to wrap this in a try / catch

Reply:
fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/205/commits/113eb8d6453e2e42feffb086697c9a2cf74df29d